hack/gen-swagger-doc/deploy.sh

```diff
@@ -30,7 +30,7 @@ Content of this repository is generated from OpenAPI specification of
 
 ## KubeVirt Containerized Data Importer API References
 
-* [master](${GITHUB_IO_FQDN}/master/index.html)
+* [main](${GITHUB_IO_FQDN}/main/index.html)
```

Comment:
This repo has not been converted yet, so if we make this change it will fail to push the api reference.